lxd/patches.go

```diff
@@ -1518,4 +1519,53 @@ func patchRemoveBackupsImagesSymlinks(_ string, d *Daemon) error {
 	return nil
 }
 
+func moveStorage(destPath string) error {
```

Comment:
can we define this as a function inside `patchMoveBackupsImagesStorage` so its scope is limited to that path?